tests/test_devices.py

```diff
@@ -60,6 +60,36 @@ def test_device_constructor_with_config(client):
     assert device.inputs_wait == {}
 
 
+def test_item_query_without_status(alarm_device):
+    """Verify that querying items without specifying a status works correctly"""
+    alarm_device.connect("username", "password")
+    # Test
+    alarm_device.update()
+    assert dict(alarm_device.items(q.SECTORS)) == alarm_device._inventory.get(q.SECTORS)
+
+
+def test_item_query_with_status(alarm_device):
+    """Verify that querying items with specifying a status works correctly"""
+    alarm_device.connect("username", "password")
+    # Test
+    alarm_device.update()
+    items = {2: {"element": 3, "excluded": False, "id": 3, "index": 2, "name": "S3 Outdoor", "status": False}}
+    assert dict(alarm_device.items(q.SECTORS, status=False)) == items
+
+
+def test_item_query_without_inventory(alarm_device):
+    """Verify that querying items without inventory populated works correctly"""
+    alarm_device._inventory = {q.SECTORS: {}, q.INPUTS: {}, q.ALERTS: {}}
```

Comment:
This initialization shouldn't be `alarm_device._inventory = {}`? I think we did that in previous tests no?